src/tasks/sep/038-world-sep-u13-to-u16a/config.toml

```
@@ -0,0 +1,53 @@
+templateName = "OPCMUpgradeV220toV410"
+
+OPCMUpgradeV220_PRESTATE = "0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9"
+OPCMUpgradeV300_PRESTATE = "0x03ee2917da962ec266b091f4b62121dc9682bb0db534633707325339f99ee405"
+OPCMUpgradeV300_UPDATE_PRESTATE = "0x03682932cec7ce0a3874b19675a6bbc923054a7b321efc7d3835187b172494b6"
+
+[[l2chains]]
+chainId = 4801
+name = "World Chain Sepolia Testnet"
+
+[[opcmUpgrades]]
+chainId = 4801
+# prestates."1.6.1-rc.1
+# https://github.com/ethereum-optimism/superchain-registry/blob/d82a61168fd1d7ef522ed8e213ce23c853031495/validation/standard/standard-prestates.toml#L6
+absolutePrestate = "0x03eb07101fbdeaf3f04d9fb76526362c1eea2824e4c6e970bdb19675b72e4fc8"
+expectedErrorsV220 = "PROXYA-10,SYSCON-20,DF-30,PDDG-50,PDDG-DWETH-30,PDDG-DWETH-40,PDDG-ANCHORP-40,PDDG-120,PLDG-10"
+# PROXYA-10 : ProxyAdmin owner is not set to L1 PAO multisig.
+# SYSCON-20 : SystemConfig gas limit is not set to 60,000,000.
+# DF-30 : DisputeGameFactory owner is not set to L1 PAO multisig.
+# PDDG-50 : Permissioned dispute game VM address mismatch.
+# PDDG-DWETH-30 : Permissioned dispute game delayed WETH owner mismatch.
+# PDDG-DWETH-40 : Permissioned dispute game delayed WETH delay not set to 1 week.
+# PDDG-ANCHORP-40 : Permissioned dispute game anchor state registry root hash mismatch.
+# PDDG-120: Permissioned dispute game challenger address mismatch.
+# PLDG-10 : Permissionless dispute game implementation not found.
+expectedErrorsV300 = "PROXYA-10,DF-30,PDDG-DWETH-30,PDDG-ANCHORP-40,PDDG-120,PLDG-10"
+# PROXYA-10 : ProxyAdmin owner is not set to L1 PAO multisig.
+# DF-30 : DisputeGameFactory owner is not set to L1 PAO multisig.
+# PDDG-DWETH-30 : Permissioned dispute game delayed WETH owner mismatch.
+# PDDG-ANCHORP-40 : Permissioned dispute game anchor state registry root hash mismatch.
+# PDDG-120: Permissioned dispute game challenger address mismatch.
+# PLDG-10 : Permissionless dispute game implementation not found.
+expectedErrorsV410 = "PROXYA-10,DF-30,PDDG-DWETH-30,PDDG-130,PLDG-10"
+# PROXYA-10 : ProxyAdmin owner is not set to L1 PAO multisig.
+# DF-30 : DisputeGameFactory owner is not set to L1 PAO multisig.
+# PDDG-DWETH-30 : Permissioned dispute game delayed WETH owner mismatch.
+# PDDG-130: The challenger is not the standard one.
+# PLDG-10 : Permissionless dispute game implementation not found.
+
```

As of v4 these two addresses should be the same, let's try that and see if CI still passes:


```suggestion
StandardValidatorV410 = "0x3bb6437aba031afbf9cb3538fa064161e2bf2d78"
```